src/game_event.cpp

```diff
@@ -625,3 +626,7 @@ const RPG::SaveMapEvent& Game_Event::GetSaveData() {
 
 	return *data();
 }
+
+bool Game_Event::IsMoveRouteActive() const {
+	return true;
```

Comment:
> Move route refactors

Why is this always true for events?

Reply:
It may be badly named, but I believe all the cases where its checked are always true for events but only true for player/vehicles with an active move route.

* Move route cancel - repeat logic - events can have custom move type so we need to check this
* `SetMaxStopCountForStep()` in `Game_Character::Move()` - This value is not set when the player moves via direction key. But is set for any event movement.
* `ForceMoveRoute()` - Need to clean up for events with custom move type.

This could use a refactor.